tedana/workflows/tedana.py

```diff
@@ -471,7 +475,8 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
                                                 t2s_limited, t2s_full, ref_img,
                                                 tes=tes, algorithm=tedpca,
                                                 source_tes=source_tes,
-                                                kdaw=10., rdaw=1.,
+                                                kdaw=10.,
+                                                rdaw=1.,
```

Comment:
Is there some reason kdaw and rdaw are on separate lines here? They seem logically related to me.

Reply:
No reason at all.